.ci.yaml

```diff
@@ -2387,12 +2387,15 @@ targets:
   # linux motog4 benchmark
   - name: Linux_android hello_world__memory
     recipe: devicelab/devicelab_drone
-    presubmit: false
+    presubmit: true
```

Comment:
Since presubmit: true is the default value, you can delete it.

Reply:
Ah will do!